cypress/e2e/group_test.spec.ts

```diff
@@ -46,6 +46,10 @@ describe("Group test", () => {
     );
   });
 
+  after(async () => {
+    await adminClient.deleteGroups();
+  });
```

Comment:
Since a Promise is returned here this can be shortened a bit.
```suggestion
  after(() => adminClient.deleteGroups());
```